test/e2e/fleet_test.go

```diff
@@ -300,13 +302,16 @@ func TestFleetRollingUpdate(t *testing.T) {
 						if maxUnavailable == 0 {
 							maxUnavailable = 1
 						}
-						// This difference is inevitable, also could be seen with Deployments and ReplicaSeets
+						// This difference is inevitable, also could be seen with Deployments and ReplicaSets
 						shift = maxUnavailable
+					} else {
+						shift = 1 // not sure why this solves the issue???
```

Comment:
What issue does this solve?

Reply:
Hmnn. That's odd, I thought I removed this. I was running the wrong feature gate'd test against the wrong install. Nice catch, I should be able to remove it.